components/support/utils/src/test/java/mozilla/components/support/utils/URLStringUtilsTest.kt

```diff
@@ -160,4 +160,34 @@ class URLStringUtilsTest {
         assertTrue(isSearchTerm("www.c-c-  "))
         assertTrue(isSearchTerm("3-3 "))
     }
+
+    @Test
+    fun urlPrefixHttps() {
```

Comment:
Thanks for adding these test cases. If we apply the change above you can fold them into a `toDisplayUrl` test, or keep separate but then I would adjust naming to indicate what the test does e.g. `stripUrlScheme` etc.